### PR TITLE
Update CHANGE_LOG for 0.51.2

### PR DESCRIPTION
**Pull request by stuartarchibald**
As title.

### CODE REVIEW

**Review by gmarkall — APPROVED** [review]
Looks good to me.

**Comment by esc** [comment]
I think the description is a bit long and unwieldy, how about:

```
This is a bugfix release for 0.51.1. It fixes a critical performance bug in the
control flow graph (CFG) back edge computation algorithm, which leads to
exponential time complexity when compiling use cases with certain
pathological properties.
```

**Comment by ehsantn** [comment]
I'd avoid using the word "pathological" :) something like "corner case" is much better.

**Comment by gmarkall** [comment]
> I'd avoid using the word "pathological" :) something like "corner case" is much better.

I've tried googling, checking definitions, and looking at the etymology to see why it would be better, but I can't find anything that suggests there's something wrong with using that word. Can you explain why it would be better to talk about a "corner case"?

(to me, "corner case" also appears less precise, which is why I'd picked the word "pathological" instead of "corner case" originally)

**Comment by ehsantn** [comment]
Not a big deal. Just a minor suggestion.

**Comment by sklam** [comment]
I'm preferring "pathological" over "corner case" because to me the latter sounds uncommon but I am afraid that CFG pattern is not uncommon.

File: CHANGE_LOG [hunk 1]
```diff
@@ -4,6 +4,21 @@ Version 0.52.0
 In development
 
 
+Version 0.51.2 (September 2, 2020)
+----------------------------------
+
+This is a bugfix release for 0.51.1, it fixes a critical performance bug in the
```
Comment (sklam) [inline CHANGE_LOG:10]:
```suggestion
This is a bugfix release for 0.51.1. It fixes a critical performance bug in the
```

File: CHANGE_LOG [hunk 2]
```diff
@@ -4,6 +4,21 @@ Version 0.52.0
...
+CFG back edge computation algorithm that lead to exponential time complexity
```
Comment (sklam) [inline CHANGE_LOG:11]:
```suggestion
CFG back edge computation algorithm that leads to exponential time complexity
```

**Pull request merged**